.eslintrc.yml

```diff
@@ -47,6 +56,7 @@ rules:
     - publicOnly: true
       require:
         ArrowFunctionExpression: true
+  max-classes-per-file: 'off'
```

Comment:
had to disable this to be able to define the error classes in `utils/validate-parameters`. there's possibly a more elegant solution but this doesn't seem to be a very useful rule for us anyways since mostly we're not using classes.